development/tests/test_server_requests.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from unittest.mock import patch
+from development.server_requests import (
+    get_logs,
+    send_challenge,
+)
+
+
+def mocked_requests_get(*args, **kwargs):
+    data = dict()
+    for key, value in zip(kwargs['keys'], kwargs['values']):
+        data[key] = value
+
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+    return MockResponse(data, 200)
```

Comment:
I think it is not necessary to do all of this there is a more straightforward way. We can talk about that in slack

Reply:
It was removed.